lib/numorstring/protocol.go

```diff
@@ -48,6 +48,24 @@ func ProtocolFromInt(p uint8) Protocol {
 	)
 }
 
+// ProtocolV3FromProtocolV1 creates a v3 Protocol from a v1 Protocol,
```

Comment:
Should be fine, but looks like duplicate code from @gunjan5's PR.

https://github.com/projectcalico/libcalico-go/pull/702/files#diff-02e818a8ad4f5514992c9563a94c08b9R74

Reply:
this one came first, I'll remove mine after this one is merged